docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -402,17 +402,104 @@ feasible based on the data present, it is also a reasonable approach.
 3.0 Results
 +++++++++++
 
+3.0 Results
++++++++++++
+
+Communication of results
+------------------------
+
+In this section we’ll discuss how health metric researchers can talk about health research related to 
+race and ethnicity without stigmatizing, othering, or otherwise causing harm to historically marginalized 
+racial and ethnic groups. It is important to be aware of how we talk about race and racism, 
+particularly in the field of health metrics sciences, our goal is often to uncover and scrutinize the 
+underlying factors contributing to health disparities.
+
+One helpful framework for differentiating ways of communicating about race is that of *race-based research vs. race-conscious research* 
+[Cerdena_2020]_. Race-based research, which is characterized 
+by race essentialism, the notion that race is a biological factor (rather than a sociopolitical construct). 
+The practice of using race in this way has historically been the norm in US health research, and causes harm for 
+historically marginalized and minoritized groups, thereby exacerbating the very health inequities their health 
+research may be attempting to elucidate and alleviate. For examples of outdated race-based practices in medicine, 
+and their race-conscious alternatives, see Table 1 from Cerdena et al.’s 2020 paper [Cerdena_2020]_. 
+Race-conscious research, on the other hand, calls for a shift in focus from race to racism as a key determinant of 
+illness and health.
+
+Key recommendations from other sources
+--------------------------------------
+
+Here we have collected recommendations from various experts and guides advocating for new standards of communication 
+regarding racial health disparities.
+
+1. **Use a health equity lens when framing information about health disparities** [Calanan_2023]_.
+
+    Rather than allowing dominant narratives around race, individualism, and meritocracy limit our understanding of 
+    the root causes of health inequities, it is important to center a health equity perspective in health metrics 
+    research.  An equity lens allows us to recognize that systems of power and oppression (including white supremacy) 
+    shape institutional policies and living conditions that systematically harm populations 
+    [AMA_2021]_.
+
+2. **Use person-first language, avoid unintentional blaming, and be as specific as possible about the group to which you're referring**. 
+    
+    For more elaboration on these tips, please see Advancing Health Equity: A Guide to Language, Narrative and Concepts 
+    [AMA_2021]_.
```

Comment:
It might be helpful to give a a quick example of each of these (person-first language, avoiding unintentional blaming, and being as specific as possible), e.g. just pulling some examples directly out of the reference, so the reader doesn't have to look it up if they have no idea what it means.

When I first read it, I thought it said "first-person language," which seemed confusing as I hadn't heard of that in this context, so I searched for "first-person" in the reference and couldn't find it, of course, which was even more confusing. It took me several minutes of searching before I finally figured out it was "person-first"...